src/Unity.Mathematics/Geometry/MinMaxAABB.cs

```diff
@@ -198,18 +198,6 @@ public void Encapsulate(float3 point)
             Max = math.max(Max, point);
         }
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static implicit operator MinMaxAABB(AABB aabb)
-        {
-            return new MinMaxAABB {Min = aabb.Center - aabb.HalfExtents, Max = aabb.Center + aabb.HalfExtents};
-        }
-
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static implicit operator AABB(MinMaxAABB aabb)
```

Comment:
If this is released to public you need to add:

`[Obsolete("<what happend> (RemovedAfter <today + 3 months in yyyy-mm-dd format>).")]`